maquetacion/scss/_titles.scss

```diff
@@ -0,0 +1,18 @@
+@import 'variables';
+
+.title {
+  align-items: center;
+  display: flex;
+  font-size: $text-big;
+  height: $title-decoration-height;
+  margin-bottom: $text-medium;
+  padding-left: 52px;
+  position: relative;
+
+  @at-root #{&}-image {
```

Comment:
I think this is not necessary. You can do it more simple with a img and a h1 tags.